portfolio/src/main/java/com/google/sps/servlets/Server2.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.servlets;
+
+
+import java.io.IOException;
+import com.google.gson.Gson;
+import java.util.ArrayList;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/head")
+public class Server2 extends HttpServlet {
```

Comment:
The name of this class should explain a little more what the class is doing. Is it serving a list of strings? Or is it going to be doing something else in the future?